test/unit/export/dataProvider/dataFormatter/RdfDataFormatterTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoSyncServer\test\unit\export\dataProvider\dataFormatter;
+
+use core_kernel_classes_ContainerCollection;
+use core_kernel_classes_Resource;
+use core_kernel_classes_Triple;
+use oat\generis\test\TestCase;
+use oat\taoSyncServer\export\dataProvider\dataFormatter\RdfDataFormatter;
+
+class RdfDataFormatterTest extends TestCase
+{
+    public function testFormat()
```

Comment:
This test doesn't cover important cases like when we have to load not standard resource classes or exclude some properties.